homeassistant/util/package.py

```
@@ -27,17 +28,22 @@ def install_package(package: str, upgrade: bool=True,
         if check_package_exists(package, target):
             return True
 
-        _LOGGER.info("Attempting install of %s", package)
+        _LOGGER.info('Attempting install of %s', package)
+        env = os.environ.copy()
         args = [sys.executable, '-m', 'pip', 'install', '--quiet', package]
         if upgrade:
             args.append('--upgrade')
-        if target:
-            args += ['--target', os.path.abspath(target)]
-
         if constraints is not None:
             args += ['--constraint', constraints]
-
-        process = Popen(args, stdin=PIPE, stdout=PIPE, stderr=PIPE)
+        if target:
+            # This only works if not running in venv
```

Sounds good. Just let it blow up?